.github/workflows/frontend-release.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish frontend components
+
+on:
+  push:
+    branches: 
+      - main
+    paths:
+      - 'projects/frontend/data-pipelines/**'
+      - 'projects/frontend/shared-components/**'
+      - '!projects/frontend/shared-components/gui/package.json'
+      - '!projects/frontend/shared-components/gui/projects/shared/package.json'
+
+jobs:
+  build-test-publish-vdk-shared:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+      with:
+        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Not sure if the GITHUB_TOKEN has write permissions to the repo. I don't have the correct access level to change it if it doesn't.